overlord/devicestate/devicemgr.go

```diff
@@ -243,10 +249,13 @@ func (m *DeviceManager) ensureSeedYaml() error {
 		return nil
 	}
 
-	coreInfo, err := snapstate.CoreInfo(m.state)
-	if err == nil && coreInfo.Name() == "ubuntu-core" {
-		// already seeded... recover
-		return m.alreadyFirstbooted()
+	if !release.OnClassic {
+		// XXX: drop this old repair code?
```

Comment:
Yeah, I think it's safe by now. The final Ubuntu Core was already "core", right?  So this should be just very old test systems which would be fine to break by now.